rust/openvas/src/openvas_redis.rs

```diff
@@ -62,50 +74,37 @@ pub trait KbAccess {
 impl KbAccess for RedisHelper<RedisCtx> {
     /// Provide access to the cache
     fn kb_id(&self) -> RedisStorageResult<u32> {
-        let cache = Arc::as_ref(&self.task_kb)
-            .lock()
-            .map_err(|e| DbError::SystemError(format!("{e:?}")))?;
-        Ok(cache.db)
+        // TODO: Should this really be self.lock_task_kb? This seems
```

Comment:
@jjnicola can you look into that?

Reply:
It is ok as it is. Just the method documentation is wrong/confusing. It returns the redis namespace ID of the current host kb that must be stored as running. Cache is related to the feed, where the vts metadata is stored. So, feel free to update the functions docs